java-extensions/hadoop-ext/src/main/java/com/starrocks/connector/share/iceberg/IcebergAwsClientFactory.java

```diff
@@ -200,6 +202,35 @@ public S3Client s3() {
         return s3ClientBuilder.build();
     }
 
+    @Override
+    public S3AsyncClient s3Async() {
```

Comment:
why need this?

Reply:
Iceberg interface require implement this. refer to https://github.com/apache/iceberg/commit/3dba6afb789a420373e44ac29ecdef866bd7ebee

Reply:
I provice a basic implementation for s3AsyncClient. Will support CRT S3AsyncClient in the future.